pkg/cmd/service/service_test.go

```diff
@@ -417,12 +408,12 @@ func TestHTTPServerWithCORS(t *testing.T) {
 func TestBuildServerWithOIDCAuthentication(t *testing.T) {
 	const localOIDCServerURL = "http://localhost:8083"
 
-	config, err := GetServiceConfig()
+	config, err := DefaultConfigWithRandomPorts()
 	require.NoError(t, err)
 
 	config.Authn.Method = "oidc"
 	config.Authn.AuthnOIDCConfig = &AuthnOIDCConfig{
-		Audience: openFGAServerAddr,
+		Audience: "openfga.dev",
```

Comment:
this doesn't change anything right?

Reply:
openFGAServerAddr isn't static anymore and this happens before the server starts up, because this config is a dependency of the server. So I just hardcoded this since it's just an audience string anyways.